snippets/ram-console/ram-console.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable Console
+# CONFIG_UART_CONSOLE is not set
```

Comment:
```suggestion
```
Don't need a line saying not set, if you want to turn it off you need to have it `=n` here

Reply:
I want to disable UART console, will update it to `=n`. Thanks!